packages/database/src/index.js

```diff
@@ -0,0 +1,61 @@
+const sqlite3 = require('sqlite3').verbose();
+
+exports.initDB =  function initDB(path) {
+    return new Promise((resolve,reject)=> {
+        resolve(new sqlite3.Database(path))
+        if(reject) {
```

Comment:
A couple of points

1. I do not think this use of promises is correct. See example here:
```javascript
return new Promise((resolve, reject) => {
   if(somethingSuccesfulHappened) {
      ...
      resolve(successObject); 
   } else {
     ...
     reject(errorObject);
   }
});
```

2. I do not think you need the promises in this file. i.e.
```javascript
exports.initDB = function initDB(path) {
  return new sqlite3.Database(path);
};
```
Same for others (can remove `await` where no promise). 
`getUsersByName` seems like the only one with a good use for promises.

Reply:
Thanks to you, I think I understand how to use Promise!!